pages/attacks/Regular_expression_Denial_of_Service_-_ReDoS.md

```diff
@@ -75,11 +75,10 @@ A Regex pattern is called **Evil Regex** if it can get stuck on crafted input.
 
 **Examples of Evil Regex**:
 
-- `(a+)+`
-- `([a-zA-Z]+)*`
-- `(a|aa)+`
-- `(a|a?)+`
-- `(.*a){x} for x \> 10`
```

Comment:
I think maybe this should have been

```
`(.*a){x}\` for x > 10
```
?

But then also the 10 is a bit arbitrary?